website/src/pages/Projects.tsx

```diff
@@ -0,0 +1,343 @@
+import { useState, useEffect } from "react";
+import { projects, cases, challenges, Project } from "@/constants/projects";
+import { ProjectDialog } from "@/components/ProjectDialog";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { useParams, useNavigate } from "react-router-dom";
+import { Trophy, Award, Lock } from "lucide-react";
+import Navbar from "@/components/Navbar";
+import Footer from "@/components/Footer";
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
+import { MAIN_SPONSOR, CASE_SPONSORS, CHALLENGE_SPONSORS, TECH_SPONSORS, FOOD_SPONSORS, VENUE_SPONSORS } from "@/constants/partners";
+
+// Define the order of cases
+const caseOrder = ['trade-republic', 'avi', 'beam'] as const;
+
+// Define the order of challenges
+const challengeOrder = ['mistral', 'tanso', 'beyond-presence', 'visionaries'] as const;
```

Comment:
You can also easily change that order in the constants files - no hard feelings. Also no `as const` needed